package.json

```diff
@@ -190,5 +190,8 @@
     "watch:test": "ava --watch",
     "watch:ts": "npm run build:ts -- --watch"
   },
-  "version": "0.16.0"
+  "version": "0.16.0",
+  "optionalDependencies": {
```

Comment:
@molant Can you move this after `nyc`, to keep things in alphabetical order? Thanks!